docs/dev-guides/personal-environments.md

```diff
@@ -469,6 +469,128 @@ gke-terra-integration-default-v2-95d71f95-gdfy   Ready    <none>   30h   v1.17.1
 
 </details>
 
+### Running Resiliency Tests Through Test Runner Library
+Test Runner Library provides the underlying framework for running resiliency tests within namespaces. The whole process involves only a few simple setup and configuration steps described below.
+
+At a high level, running resiliency tests within namespaces require a set of permissions to manipulate cluster resources with Kubernetes API.
+These permissions are namespace scoped so that no resiliency tests will have cluster-wide access.
+
+The required namespace permissions are specified in the 3 manifest templates which comes with the Test Runner Library distribution.
+The `setup-k8s-testrunner.sh` script templates the formation of the actual manifests for deploying to a namespace.
+The `setup-k8s-testrunner.sh` script also carries out the following functions:
+
+* Provision the Kubernetes Service Account, RBAC Role and RoleBinding for Test Runner.
+* Export credentials of the Test Runner Kubernetes Service Account to Vault.
+
+To set up a namespace for Test Runner resiliency tests, simply run the command in the following example (`terra-zloery` namespace for example).
+
+The first argument is the `kubectl context` mentioned elsewhere in this document.
+
+The second argument is the Terra namespace (without the `terra-` prefix).
+
+The third argument is just some text to describe the application itself.
+```shell script
+$ ./setup-k8s-testrunner.sh gke_terra-kernel-k8s_us-central1-a_terra-integration zloery workspacemanager
+```
+
+Once the script above ran successfully, the namespace is ready for resiliency testing through the Test Runner Framework.
+
+The Kubernetes credentials stored in Vault needs to be rendered by means of the `./render-k8s-config.sh zloery` script in the test repository before kicking off the resiliency tests.
+```shell script
+$ ./render-k8s-config.sh zloery
+```
+
+<summary>testrunner-k8s-serviceaccount.yml.template</summary>
```

Comment:
I don't think the summary node shows up unless it's inside a `<details>` block

Reply:
Updated this one to include `<details>` block